language/move-lang/src/parser/mod.rs

```diff
@@ -112,21 +114,16 @@ fn ensure_targets_deps_dont_intersect(
     ))
 }
 
-// TODO replace with some sort of intern table
```

Comment:
The prophecy fulfilled!

Reply:
I wonder how old this TODO is lol, its probably at least from Summer 2019 if not earlier...

Reply:
First added in 837d86fb74fdc97854de4b64f4bd9ad9d219163a, from January 28 2020! 😮